pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -295,8 +295,8 @@ def validation_step(self, *args, **kwargs):
     def test_step(self, *args, **kwargs):
         return self.lightning_module.test_step(*args, **kwargs)
 
-    def predict(self, *args, **kwargs):
-        return self.lightning_module.predict(*args, **kwargs)
+    def predict_step(self, *args, **kwargs):
```

Comment:
I see the point with consistency to `train_step`, but would it be closer to what it does as `predict_batch`

Reply:
Yes, it is for consistency for other functions which also operate on batch.